concepts/functions/about.md

```diff
@@ -170,49 +198,56 @@ Now, we can call the functions in the order we want.
 
 ## Scope of Variables
 
-If variable is defined inside a function, then it will be only accessible inside the function. If we want to access the variable outside the function, we need to use the [`global`][global] keyword. [`nonlocal`][nonlocal] keyword is used to access the variable inside a nested function.
+If variable is defined inside a function, then it will be only accessible inside the function.
+If we want to access the variable outside the function, we need to use the [`global`][global] keyword.
+[`nonlocal`][nonlocal] keyword is used to access the variable inside a nested function.
```

Comment:
This might need a little more elaboration, since this is about referencing enclosing scope that's _not_ global, but not exactly local either and may or may not be important when making closures and other nested functions and structures.

Reply:
Yeah, this area isn't my strong suit, as I've never needed to use either `global` or `nonlocal`. But my experience is pretty much limited to the Exercism exercises.

Anyway, this is from the original about.md which was originally with this verbiage long ago.

Reply:
Well, the original was "optimistically merged".  😄   But when I get a chance, I'll propose something.  I don't know that we need much ...  but maybe a few sentences and an example.